scripts/esbuild/build.ts

```diff
@@ -8,14 +10,32 @@ import { buildScreenshot } from './screenshot';
 import { buildSysNode } from './sys-node';
 import { buildTesting } from './testing';
 
-// the main entry point for the Esbuild-based build
-async function main() {
+// the main entry point for the build
+export async function run(rootDir: string, args: ReadonlyArray<string>) {
   const opts = getOptions(process.cwd(), {
-    isProd: !!process.argv.includes('--prod'),
-    isCI: !!process.argv.includes('--ci'),
-    isWatch: !!process.argv.includes('--watch'),
+    isProd: !!args.includes('--prod'),
+    isCI: !!args.includes('--ci'),
+    isWatch: !!args.includes('--watch'),
   });
 
+  try {
+    if (args.includes('--release')) {
+      await release(rootDir, process.argv);
+      return;
+    }
+
+    if (process.argv.includes('--validate-build')) {
+      await validateBuild(rootDir);
+      return;
+    }
+    await buildAll(opts);
+  } catch (e) {
+    console.error(e);
+    process.exit(1);
+  }
+}
```

Comment:
this is added in here based on the code in `run` in `scripts/build.ts` and calls the `release` and `validateBuild` functions in the same way